.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# Description
+Please include a summary of the changes and the related issue. List any dependencies that are required for this change.
+
+# Linked Items:
+Please include the associated items of this Pull Request using the following syntax to link them for closure on completion of this Pull Request.
+`Closes #<Issue Number>`
+`Fixes #<Issue Number>`
+`Resolves #<Issue Number>`
+
+## Type of change
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Feature improvement
+- [ ] Requires a documentation update
```

Comment:
We may want to start requiring documentation updates for most changes, except possibly minor bug fixes, etc so this may need to be an automatic assumption.

Reply:
Can remove the `Requires documentation update` from the type of change section and leave it in the final check list about `Updated documentation accordingly` as a reminder.

Reply:
I think it's fine to leave as a reminder to check for documentation updates.